content/_data/authors/dheerajodha.adoc

```diff
@@ -0,0 +1,5 @@
+---
+name: "Dheeraj Singh Jodha"
+github: dheerajodha
+linkedin: dheeraj-singh-jodha
+---
```

Comment:
Would be nice to attach a smaller avatar. 1.2 Mb is a pretty big file

Reply:
Done, thanks